compiler/lowerings.nim

```diff
@@ -60,7 +60,6 @@ proc lowerTupleUnpacking*(g: ModuleGraph; n: PNode; owner: PSym): PNode =
   var temp = newSym(skTemp, getIdent(g.cache, genPrefix), owner, value.info, g.config.options)
   temp.typ = skipTypes(value.typ, abstractInst)
   incl(temp.flags, sfFromGeneric)
-  incl(temp.flags, sfCursor)
```

Comment:
Awww, still not correct? :-)

Reply:
IMO, it can't be correct because following example can't work if every unpacked tuple is cursor:
```nim
proc xx(arg: tuple[XX]) = 
  let (x, y) = arg   # arg needs to be copied here
```

Reply:
Bad example, but fair enough.